src/lib/marshal/workflow/helpers.ts

```diff
@@ -104,6 +110,8 @@ const batchStepSummaryLines = (step: WorkflowStepData) => {
     batch_key && `Batch key: ${batch_key}`,
     duration && `Batch window: ${duration.value} ${duration.unit}`,
     field_path && `Batch window: "${field_path}"`,
+    batch_window_type &&
+      `Batch duration is fixed: ${batch_window_type === BatchWindowType.Fixed}`,
```

Comment:
As for the display itself, I might just show the value itself transparently? i.e. `Batch window type: fixed | sliding`. 

As is now, "Batch duration is fixed: false" feels somewhat opaque to me, because it's not clear immediately what it means to be "false".

Reply:
Will fix the typing check to make it `"sliding" | "fixed"`

My goal here was to show something similar to what we show on the dashboard!
On the dashboard we don't display the sliding/fixed to the user, it's more of an internal concept.
On the batch step manage settings we display a toggle that reads: "Extend batch duration when new activities are added."

My only concern about showing the `fixes` and `sliding` is I think there is a chance of customers not understanding what we are referring to.

Reply:
Ah gotcha, thanks for that context.

Then maybe another alternative is to massage the messaging to match it more closely? i.e. 
"Batch duration: fixed" vs "Batch duration: extended with new activities"

Having said that, "fixed" vs "sliding" window feels somewhat a familiar way to express for developers, but I think either approach works!